### PR TITLE
Add more tests for cardinality HLL Presto function

### PR DESCRIPTION
**Pull request by mbasmanova**
Add unit tests specifically for cardinality(hll) Presto function in addition to tests in velox/functions/prestosql/aggregates/tests/ApproxDistinctTest.cpp.

### CODE REVIEW

File: velox/functions/prestosql/tests/HyperLogLogFunctionsTest.cpp [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+namespace {
+
+template <typename T>
+uint64_t hashOne(T value) {
+  return XXH64(&value, sizeof(value), 0);
+}
+
+class HyperLogLogFunctionsTest : public functions::test::FunctionBaseTest {
+ protected:
+  std::string serialize(int8_t indexBitLength, const SparseHll& sparseHll) {
```
Comment (amitkdutta) [inline velox/functions/prestosql/tests/HyperLogLogFunctionsTest.cpp:35]: Lets make this method const 

`std::string serialize(int8_t indexBitLength, const SparseHll& sparseHll) const`

File: velox/functions/prestosql/tests/HyperLogLogFunctionsTest.cpp [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    std::string serialized;
+    serialized.resize(sparseHll.serializedSize());
+    sparseHll.serialize(indexBitLength, serialized.data());
+    return serialized;
+  }
+
+  std::string serialize(DenseHll& denseHll) {
```
Comment (amitkdutta) [inline velox/functions/prestosql/tests/HyperLogLogFunctionsTest.cpp:42]: Same

**Review by amitkdutta — APPROVED** [review]
Looks good. Minor Nits.

File: velox/functions/prestosql/tests/HyperLogLogFunctionsTest.cpp (hunk 1 above)
Comment (mbasmanova) [reply velox/functions/prestosql/tests/HyperLogLogFunctionsTest.cpp:35]: Actually, I'll make it static.

**Pull request closed without merging**